examples/DotNetCore/SimpleConsoleApp/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.SimpleConsoleApp
+{
+    using Microsoft.Extensions.Configuration;
+
+    class Program
+    {
+        static void Main(string[] args) {
```

Comment:
Can we create the project using the latest template that doesn't have the main function etc?

Reply:
Updated.